lib/versioning/composer/index.spec.ts

```diff
@@ -127,6 +127,14 @@ describe('semver.getNewValue()', () => {
         toVersion: 'V1.1',
       })
     ).toEqual('V1.1');
+    expect(
+      semver.getNewValue({
+        currentValue: '^1.0',
+        rangeStrategy: 'pin',
+        fromVersion: '1.0',
+        toVersion: 'V1.1',
```

Comment:
Is it likely that a registry would return a version of `V1.1` and not `1.1.0`? But if so, this expected value is correct.

Reply:
You're right. I copied this from an existing test and just kept it the same. The test was just a check if caret works with the same conditions as tilde.